workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentDataGenerator.tt

```diff
@@ -19,7 +20,40 @@ namespace <#= qualifiedNamespace #>
         {
             public uint ComponentId => <#= componentDetails.ComponentId #>;
 
-            public BlittableBool DirtyBit { get; set; }
+            [Flags]
+            public enum DirtyBitsEnum
+            {
+                None = 0,
+<# int currentFieldBinaryIdx = 1; #>
```

Comment:
If you change to using a normal for loop, you can use `i +1` instead of having a seperate variable

Reply:
Done